assets/src/components/cluster/Metadata.tsx

```diff
@@ -0,0 +1,69 @@
+import { Card, Chip } from '@pluralsh/design-system'
+import { Div, Flex } from 'honorable'
+import { ReactNode } from 'react'
+import type { Metadata as MetadataT } from 'generated/graphql'
+
+export const mapify = tags => tags.reduce((acc, { name, value }) => ({ ...acc, [name]: value }), {})
+
+export function MetadataRow({
+  name,
+  children,
+}: {
+  name: ReactNode
+  children: ReactNode
+}) {
+  return (
+    <Div>
+      <Div
+        body1
+        bold
+        marginBottom="small"
+      >
+        {name}
+      </Div>
+      <Flex
+        direction="row"
+        wrap
+        gap="xsmall"
+      >
+        {children}
+      </Flex>
+    </Div>
+  )
+}
+
+export function MetadataTag({ name }) {
+  return <Chip>{name}</Chip>
+}
+
+export function Metadata({
```

Comment:
We should probably merge it at some point with what I've added to `ComponentInfoMetadata`.